draft-smyslov-ipsecme-ikev2-mceliece-00.xml

```diff
@@ -101,23 +103,41 @@
             group offered in an ADDKE payload if the initiator has not negotiated the support for both big
             payloads and for fragmentation.</t>
           <t> Within the IKE_INTERMEDIATE exchange, such payloads will normally encode the large public
-            keys. A diagram is provided here for illustration:</t>
+            keys. A diagram is provided here for illustration:. The format below is the big payload version
+            of the Key Exchange from section 3.4 of <xref target="RFC7296"/>.</t>
          <figure> 
             <artwork><![CDATA[
                         1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
    | Next Payload  |C|L| RESERVED  |         Payload Length        
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-      continued.                   | The public key of the McEliece
+      length continued...          |   Diffie-Hellman Group Num    | 

```

Comment:
This field is renamed to "Key Exchange Method" by RFC 9370. I think we should use its current name..